zookeeper-server/src/test/java/org/apache/zookeeper/test/EnforceQuotaTest.java

```diff
@@ -54,14 +56,16 @@ public void tearDown() throws Exception {
 
     @Test
     public void testSetQuotaDisableWhenExceedBytesHardQuota() throws Exception {
-        final String path = "/c1";
+        final String namespace = UUID.randomUUID().toString();
```

Comment:
Why not `namespace = "c1"`?  Is there a specific reason to change from a static path to a dynamic/random one?

(I haven't looked into the tests yet, so may be missing something obvious.)

Reply:
Yes, the reason is that we added validation on the per-namespace quota exceeded count in the test cases. The namespace need to be dynamic/random to make sure the counter is not interfered by other test cases.

`ZooKeeperQuotaTest.validateNoQuotaExceededMetrics(namespace);`